### PR TITLE
Project structure changed according to clean architecture guide

### PR DESCRIPTION
**Pull request by FrancoKain**
Implementation of clean architecture guidelines

### CODE REVIEW

File: lib/src/presentation/widgets/movie_image_from_movie_list.dart [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 
-import '../utils/styles.dart';
+import '../../core/utils/styles.dart';
 
 class MovieImageFromMovieList extends StatelessWidget {
```
Comment (Kevin-Guillen) [inline lib/src/presentation/widgets/movie_image_from_movie_list.dart:5]: I would rename this to something like MoviesListMovieImage

File: lib/src/presentation/widgets/movie_information_from_movie_list.dart
Comment (Kevin-Guillen) [inline lib/src/presentation/widgets/movie_information_from_movie_list.dart:1]: i would rename this to something like movie_list_information

File: lib/src/presentation/widgets/movie_image_from_movie_list.dart
Comment (Kevin-Guillen) [inline lib/src/presentation/widgets/movie_image_from_movie_list.dart:1]: i would rename this to something like movie_list_image

File: lib/src/presentation/widgets/movie_information_from_movie_list.dart
Comment (FrancoKain) [reply lib/src/presentation/widgets/movie_information_from_movie_list.dart:1]: Done

**Review by matiasilveiro-gl — APPROVED** [review]
The overall architecture looks good!

File: lib/src/presentation/widgets/movie_image_from_movie_list.dart
Comment (FrancoKain) [reply lib/src/presentation/widgets/movie_image_from_movie_list.dart:1]: Done

**Review by Kevin-Guillen — APPROVED** [review]
lgtm

**Review by MarcosCGlobant — APPROVED** [review]

**Pull request merged**